.github/workflows/ci-project.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           - 'ia32-generic-qemu'
           - 'riscv64-generic-qemu'
           - 'riscv64-generic-spike'
+          - 'riscv64-noelv-fpga'
```

Comment:
:exclamation: this is minor addition, but might impact CI greatly - our usage tier has job concurrency limit == 20, this is 21st job in the matrix (so - some job might be blocked).

IMHO you can merge this now, we probably should think about making our CI explicitly multi-stage (eg. start with building "first-class" targets, than, "second-class", than testing)

Reply:
Thanks for the info, if so, then I'm fine with removing `riscv64-generic-spike` target in favour of the `noelv-fpga` for the time being, especially since we have the Zynq Ultrascale port also coming, which will be completely new architecture.

I agree that we should probably introduce some changes to our CI

Reply:
@damianloew just FYI, no need to improve our CI right away.